plugin/go.mod

```diff
@@ -3,32 +3,32 @@ module github.com/hashicorp/go-kms-wrapping/plugin/v2
 go 1.17
```

Comment:
I believe this should be bumped to 1.20 to match the others? I'm getting an error from a `go mod tidy` run in the plugin folder until I bump it

```
go mod tidy
github.com/hashicorp/go-kms-wrapping/plugin/v2 imports
	github.com/stretchr/testify/require imports
	github.com/stretchr/testify/assert imports
	gopkg.in/yaml.v3 tested by
	gopkg.in/yaml.v3.test imports
	gopkg.in/check.v1 loaded from gopkg.in/check.v1@v1.0.0-20190902080502-41f04d3bba15,
	but go 1.16 would select v1.0.0-20201130134442-10cb98267c6c
github.com/hashicorp/go-kms-wrapping/plugin/v2 imports
	github.com/stretchr/testify/require imports
	github.com/stretchr/testify/assert imports
	gopkg.in/yaml.v3 tested by
	gopkg.in/yaml.v3.test imports
	gopkg.in/check.v1 imports
	github.com/kr/pretty imports
	github.com/rogpeppe/go-internal/fmtsort loaded from github.com/rogpeppe/go-internal@v1.6.1,
	but go 1.16 would select v1.9.0

To upgrade to the versions selected by go 1.16:
	go mod tidy -go=1.16 && go mod tidy -go=1.17
If reproducibility with go 1.16 is not needed:
	go mod tidy -compat=1.17
For other options, see:
	https://golang.org/doc/modules/pruning
```

Reply:
ty!